www/.gitignore

```diff
@@ -33,4 +33,4 @@ src/cache
 /package-lock.json
 
 # Env variables
-.env.development
+.env.*
```

Comment:
Maybe `.env*` so it'll catch the base case, too? e.g. `.env`

Reply:
Great idea, pushing that change now.